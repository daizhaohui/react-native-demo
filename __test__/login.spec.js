import Login from "../src/components/login/";
import React from "react";
import renderer from "react-test-renderer";

it("renders", () => {
  const r = renderer.create(<Login />).toJSON();
  expect(r).toMatchSnapshot();
});
