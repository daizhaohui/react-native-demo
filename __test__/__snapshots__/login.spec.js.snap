// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  style={
    Object {
      "flexDirection": "column",
      "height": 200,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "width": 100,
        }
      }
    >
       用户名: 
    </Text>
    <TextInput
      allowFontScaling={true}
      maxLength={30}
      onChangeText={[Function]}
      placeholder="请输入用户名"
      style={
        Object {
          "borderRadius": 4,
          "borderWidth": 1,
          "height": 40,
          "width": 200,
        }
      }
      textContentType="name"
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "width": 100,
        }
      }
    >
       密码: 
    </Text>
    <TextInput
      allowFontScaling={true}
      maxLength={20}
      onChangeText={[Function]}
      placeholder="请输入密码"
      style={
        Object {
          "borderRadius": 4,
          "borderWidth": 1,
          "height": 40,
          "width": 200,
        }
      }
      textContentType="password"
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      accessibilityLabel="登录"
      accessibilityRole="button"
      accessibilityStates={Array []}
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          登  录
        </Text>
      </View>
    </View>
  </View>
</View>
`;
